data/nation/texan.ruleset

```diff
@@ -5,28 +5,35 @@ plural=_("?plural:Texans")
 groups="American"
 legend=_("Texas, \"the Lone Star State\" is the second largest and second\
  most populous state of the USA, famous for its \"larger than life\"\
- cowboy mentality. During its history, Texas has been ruled in full or in\
- part by six countries: France, Spain, Mexico, the Texas Republic, the\
- Confederacy and the USA - as expressed in the slogan \"six flags over\
- Texas\".")
+ cowboy culture and Spanish heritage. After the famous Battle of the Alamo de Bexar,\
+ The Texians defeated the Mexicans at San Jacinto and declared independence.\
+ The Republic of Texas lasted from 1836 to 1846, after which it became the\
+ 28th state of the United States. Historically, Texas was ruled in full or in\
+ part by six nations: France, Spain, Mexico, the Republic of Texas, the\
+ Confederacy, and the USA, as expressed in the slogan \"six flags over Texas\".")
 
 leaders = {
  "name",                "sex"
- "Stephen F. Austin",   "Male"
- "Juan N. Seguin",      "Male"
- "Sam Houston",         "Male"
- "Mirabeau B. Lamar",   "Male"
- "James P. Henderson",  "Male"
- "Ma Ferguson",         "Female"
- "John Connally",       "Male"
- "Lyndon B. Johnson",   "Male"
- "George W. Bush",      "Male"
+ "Stephen F. Austin",      "Male"
+ "Juan N. Seguin",         "Male"
+ "Sam Houston",            "Male"
+ "Francis Lubbock",        "Male"
+ "Mirabeau B. Lamar",      "Male"
+ "James P. Henderson",     "Male"
+ "Lawrence Sullivan Ross", "Male"
+ "Ma Ferguson",            "Female"
+ "John Connally",          "Male"
+ "Lyndon B. Johnson",      "Male"
+ "Ann Richards",           "Female"
+ "George W. Bush",         "Male"
```

Comment:
This lists doesn't seem to have any [pre-revolution leader](https://en.wikipedia.org/wiki/List_of_Texas_governors_and_presidents), so much for "six flags over Texas" :stuck_out_tongue:

Reply:
Spain is a weird one because the country didn't have one ruler at the time. I'll add in King Ferdinand V who ruled over Leon at the time, which is considered the greater kingdom of the time period. I'll add in King Louis XIV for France and Santa Anna for Mexico. Stephen F Austin is the Republic of Texas and Lawrence Ross during confederacy.